include/ros1_bridge/factory.hpp

```diff
@@ -103,9 +112,9 @@ class Factory : public FactoryInterface
     ros::Publisher ros1_pub,
     rclcpp::PublisherBase::SharedPtr ros2_pub = nullptr)
   {
-    rmw_qos_profile_t custom_qos_profile = rmw_qos_profile_sensor_data;
-    custom_qos_profile.depth = queue_size;
-    return create_ros2_subscriber(node, topic_name, custom_qos_profile, ros1_pub, ros2_pub);
+    auto qos = rclcpp::SensorDataQoS();
+    qos.keep_last(queue_size);
```

Comment:
I don't think it's any better or worse, but you can do this I think (I thought the KeepLast constructor would work)

```suggestion
    auto qos = rclcpp::SensorDataQoS(rclcpp::KeepLast(queue_size));
```

Reply:
Thanks, I updated both code locations with the recommended change: b5c4c5ccb98cd1ffc2499e341e8c90f2f67d4982.